app/styles/components/koenig-content.css

```diff
@@ -0,0 +1,416 @@
+    margin: .8rem 0 0;
+}
+
+.koenig-editor__editor h3 + p,
+.koenig-editor__editor h4 + p,
+.koenig-editor__editor h5 + p,
+.koenig-editor__editor h6 + p {
+    margin: .4rem 0 0;
+}
+
+.koenig-editor__editor p + p,
+.koenig-editor__editor blockquote + p,
+.koenig-editor__editor ul + p,
+.koenig-editor__editor ol + p
+ {
+    margin: 3.2rem 0 0;
+}
+
+.koenig-editor__editor div + p { /* Mobiledoc cards can be addressed by their wrapper div element */
+    margin: 2.8rem 0 0;
+}
+
+
+/* Lists
+/* -------------------------------------------------------------------- */
+
+/* .koenig-editor__editor ul, */
+.koenig-editor__editor ul:not(.kg-action-bar),
+.koenig-editor__editor ol {
+    padding: 0;
+    margin: 2.4rem 0 0;
+}
+
+.koenig-editor__editor ul:not(.kg-action-bar) li,
+.koenig-editor__editor ol li {
+    margin: 0.8rem 0 0 2.8rem;
+    line-height: 3.2rem;
+}
+
+.koenig-editor__editor ul:not(.kg-action-bar) li:first-child,
+.koenig-editor__editor ol li:first-child {
+    margin: 0 0 0 2.8rem;
+}
+
+
+/* Cards in general
+/* -------------------------------------------------------------------- */
+
+.koenig-editor__editor > div {
+    line-height: 0;
+}
+
+.koenig-editor__editor div + div {
+    margin: 3.2rem 0 0;
+}
+
+.koenig-editor__editor p + div,
+.koenig-editor__editor blockquote + div,
+.koenig-editor__editor ul + div,
+.koenig-editor__editor ol + div {
+    margin: 3.2rem 0 0;
+}
+
+.koenig-editor__editor h1 + div {
+    margin: 2.8rem 0 0;
+}
+
+.koenig-editor__editor h2 + div,
+.koenig-editor__editor h3 + div,
+.koenig-editor__editor h4 + div,
+.koenig-editor__editor h5 + div,
+.koenig-editor__editor h6 + div {
+    margin: 1.6rem 0 0;
+}
+
+.koenig-editor__editor hr {
+    margin: 0;
+}
+
+
+/* Links
+/* -------------------------------------------------------------------- */
+
+.koenig-editor__editor a {
+    color: var(--darkgrey);
+    text-decoration: none;
+    white-space: nowrap;
+    box-shadow: inset 0 -1px 0 var(--blue);
+}
+
+
+/* Blockquote
+/* -------------------------------------------------------------------- */
+
+.koenig-editor__editor blockquote p {
+    margin: 0;
+}
+
+.koenig-editor__editor blockquote {
+    border-left: 0.25rem solid var(--blue);
+    padding-left: 2.0rem;
+    font-style: italic;
+}
+
+.koenig-editor__editor h1 + blockquote,
+.koenig-editor__editor h2 + blockquote {
+    margin: .8rem 0 0;
+}
+
+.koenig-editor__editor h3 + blockquote,
+.koenig-editor__editor h4 + blockquote,
+.koenig-editor__editor h5 + blockquote,
+.koenig-editor__editor h6 + blockquote {
+    margin: .4rem 0 0;
+}
+
+.koenig-editor__editor p + blockquote,
+.koenig-editor__editor blockquote + blockquote {
+    margin: 3.2rem 0 0;
+}
+
+
+/* Mobiledoc cards can be selected by their wrapper div element */
+.koenig-editor__editor div + blockquote {
+    margin: 2.8rem 0 0;
+}
+
+
+/* Code
+/* ------------------------------------------------------------------ */
+.koenig-editor__editor code, pre {
```

Comment:
Is this missing a class selector in front of the `pre`?

Reply:
Fixed it.